src/llmq/signing.cpp

```diff
@@ -173,7 +174,7 @@ void CRecoveredSigsDb::WriteRecoveredSig(const llmq::CRecoveredSig& recSig)
     {
         LOCK(cs_cache);
         hasSigForIdCache.insert(std::make_pair(recSig.getLlmqType(), recSig.getId()), true);
-        hasSigForSessionCache.insert(signHash, true);
+        hasSigForSessionCache.insert(signHash.Get(), true);
```

Comment:
consider further refactoring by using `SignHash` as a type for  `hasSigForSessionCache`, for `uniqueSignHashes`, as argument in `PushMessage`, `InitSession`, as a member of `SignSession`, etc